# PhyloGWAS
Use PhyloGWAS to identify genes that are associated with floral color in *Pedicularis*.

## Basic information
PhyloGWAS is used to identify genetic variation associate with trait evolution. More specifically, PhyloGWAS tries to find **nonsynonymous variants that are shared by species with same phenotypic trait but are *not* in a monophyletic group.**

PhyloGWAS is a variant of GWAS (Genome-wide associate studies). Instead of sequencing more than 100 individuals of two populations with distinct phenotypic trait, PhyloGWAS sequences multiple species with a distinct phenotypic trait and divide these species into two groups (like "populaitons" in GWAS). It believes that these species undergo a parallell evolution under similar environmental conditions. Therefore, compared to GWAS, PhyloGWAS:
- sequences multiple species rather than individuals
- requires species that have same phenotypic trait are not in a monophyletic group

<img src="https://github.com/yuemeanshappy/phyloGWAS/blob/main/images/gwas_and_phylogwas.jpg" alt="comparision between GWAS and PhyloGWAS" class="center" width=50%>

## Procedures
1. 


## References
**papers**
1. Pease JB, Haak DC, Hahn MW, Moyle LC (2016) Phylogenomics Reveals Three Sources of Adaptive Variation during a Rapid Radiation. PLoS Biol 14(2): e1002379. doi:10.1371/journal. pbio.1002379
2. Wu et. al. (2018) Dissecting the basis of novel trait evolution in a radiation with widespread phylogenetic discordance. Molecular Ecology. 27 (16): doi:10.1111/mec.14780

**github repos**
1. wum5/JaltPhylo\
https://github.com/wum5/JaltPhylo#introgression-analysis
2. ACColin/PhyloJaltoWorkflow\
https://github.com/ACColin/PhyloJaltoWorkflow
3. peaselab/mvftools\
https://github.com/peaselab/mvftools


